lib/modules/platform/github/schema.spec.ts

```diff
@@ -82,7 +81,6 @@ describe('modules/platform/github/schema', () => {
     ]);
     expect(error).toBeUndefined();
   });
-
```

Comment:
revert

Reply:
Oops, sorry thats my VS Code issue which removed all the existing blanks. But anyways fixed it.